applications/nrf_desktop/README.rst

```diff
@@ -71,22 +71,59 @@ The nRF Desktop application is built in the same way to any other NCS applicatio
 
 .. include:: /includes/build_and_run.txt
 
-Configuration option set depends on the selected build type.
-The project supports the following types:
+Build types
+===========
 
-    * ZRelease - release version of the application with no debugging features
-    * ZDebug - the same functionality as ZRelease but with debug options enabled
-    * ZDebugWithShell - the same set as ZDebug plus shell is enabled
-    * ZReleaseMCUBoot - the same set as ZRelease but support of MCUBoot is enabled
-    * ZDebugMCUBoot - the same set as ZDebug but support of MCUBoot is enabled
+The build type is a custom feature of the nRF52 Desktop project.
+The build type enables user to select a set of configuration options without making changes to the ``.conf`` files.
 
-Build type is specified using the cmake variable CMAKE_BUILD_TYPE.
+The project supports the following build types:
+
+    * ``ZRelease`` - release version of the application with no debugging features.
+    * ``ZDebug`` - the same functionality as ZRelease but with debug options enabled.
+    * ``ZDebugWithShell`` - the same set as ZDebug plus shell is enabled.
+    * ``ZReleaseMCUBoot`` - the same set as ZRelease but support of MCUBoot is enabled.
+    * ``ZDebugMCUBoot`` - the same set as ZDebug but support of MCUBoot is enabled.
+
+Not every build type may be supported for the given board.
```

Comment:
How do I know if it is supported for my board?

Reply:
You should check if the *.config files exist for the board and build type. I will add this to documentation.

Reply:
Maybe it would be easier to rely on error log in case there is no such configuration for the board.
Configuration details will be described in porting guide.